portfolio/src/main/webapp/script.js

```diff
@@ -156,4 +156,38 @@ function deleteAll() {
         );
   })
   location.reload();
-}
\ No newline at end of file
+}
+
+function fetchBlobstoreUrlAndShowImage() {
+  fetch('/blobstore-upload-url')
+  .then((response) => {
+    return response.text();
+  })
+  .then((imageUploadUrl) => {
+    const messageForm = document.getElementById('my-form');
+    messageForm.action = imageUploadUrl;
+    messageForm.classList.remove('hidden');
+    getImageSRC();
+  });
+}
+
+function getImageSRC() {
+  fetch('/my-form-handler')
+  .then(response => response.json())
+  .then((images) => {
+    if(images.length == checkIfAnElementIsEmpty ){
```

Comment:
I think you can simply check for images.isEmpty()

Reply:
yes I forgot you can do this lol

Reply:
Nevermind you can not do this as isEmpty() is only for a string type variable and this function does not exist in javascript but in Java it does

Reply:
Ahh your right. However you may want to use Javascript's strictly equal here.

Reply:
I assuming you mean the 3 equal signs, and will do